docs/features/files.md

```diff
@@ -0,0 +1,11 @@
+# Files and volumes
```

Comment:
Do we have a follow up task to document the `copy*` API?
Since we recommend using it instead of file-based mounting, it's important to start with it in the docs. Although, can be added later

Reply:
Also, I just added one to migrate `withClasspathResourceMapping` to copy-like API:
https://github.com/testcontainers/testcontainers-java/issues/1163

Reply:
Raised #1164